lib/charms/operator_libs_linux/v2/snap.py

```diff
@@ -409,6 +412,17 @@ def unhold(self) -> None:
         """Remove the refresh hold of a snap."""
         self._snap("refresh", ["--unhold"])
 
+    def alias(self, application: str, alias: str = None) -> None:
```

Comment:
Seems like the type of `alias` should be `Optional[str]`.

Reply:
Updated.